federation/pkg/federatedtypes/scheduling.go

```diff
@@ -87,6 +93,57 @@ func (a *replicaSchedulingAdapter) IsSchedulingAdapter() bool {
 	return true
 }
 
+func isSelected(names []string, name string) bool {
```

Comment:
If you are talking about [this](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/string.go#L19), I found it less efficient then using the simple slice like I have used here, simply by the fact that it uses maps (I have used it in hpa controller though, because of the need of set operations over there).
I could have used this string set to store the cluster list in the first place to ensure, I don't introduce a function like the one, I got this comment on. But a slice suited the need better because of marshalling being super easy [as here](https://github.com/kubernetes/kubernetes/pull/49950/commits/41b6440a938c00b3b8b40881bed986b1304cdb71#diff-153c05d91ab301e028511701ea78219bR39).